examples/cd_spot_the_diff_mnist_wine.ipynb

```diff
@@ -0,0 +1,810 @@
+{
```

Comment:
This notebook to be removed as it is a duplicate (but without NME reference) as the other notebook?